# install files in /usr/share/games and /usr/lib/games
Index: opensonic-0.1.4/CMakeLists.txt
===================================================================
--- opensonic-0.1.4.orig/CMakeLists.txt	2011-04-27 22:05:51.301485000 +0200
+++ opensonic-0.1.4/CMakeLists.txt	2011-04-27 22:07:20.321485011 +0200
@@ -410,16 +410,12 @@
 
 # Installing on *nix
 IF(UNIX)
-  SET(GAME_INSTALL_DIR "/usr/share/${GAME_UNIXNAME}")
-  SET(GAME_FINAL_DIR "/usr/bin")
+  SET(GAME_INSTALL_DIR "/usr/share/games/${GAME_UNIXNAME}")
+  SET(GAME_FINAL_DIR "/usr/lib/games/${GAME_UNIXNAME}")
 
   INSTALL(CODE "MESSAGE(\"Installing ${GAME_NAME} ${GAME_VERSION}... Make sure you have root privileges.\")")
-  INSTALL(TARGETS ${GAME_UNIXNAME} RUNTIME DESTINATION ${GAME_INSTALL_DIR})
-  INSTALL(FILES license.txt readme.html DESTINATION ${GAME_INSTALL_DIR})
+  INSTALL(TARGETS ${GAME_UNIXNAME} RUNTIME DESTINATION ${GAME_FINAL_DIR})
   INSTALL(DIRECTORY objects sprites config images levels licenses musics quests samples screenshots themes languages DESTINATION ${GAME_INSTALL_DIR} PATTERN ".svn" EXCLUDE)
 
-  INSTALL(CODE "MESSAGE(\"Creating files at ${GAME_FINAL_DIR}...\")")
-  INSTALL(CODE "EXECUTE_PROCESS(COMMAND \"cmake\" \"-E\" \"copy\" \"${GAME_INSTALL_DIR}/${GAME_UNIXNAME}\" \"${GAME_FINAL_DIR}/${GAME_UNIXNAME}\")")
-
   INSTALL(CODE "MESSAGE(\"Done! Please run ${GAME_UNIXNAME} to start ${GAME_NAME}.\")")
 ENDIF(UNIX)
